config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -1320,7 +1320,7 @@ public HttpSecurity authorizeRequests(
 	 * &#064;Configuration
 	 * &#064;EnableWebSecurity
 	 * public class AuthorizeUrlsSecurityConfig extends WebSecurityConfigurerAdapter {
-	 *
+	 *HttpSecurity.java
```

Comment:
It looks like there's some stray text that got pasted here. Would you please remove it?